src/utils/LoginUtils.ts

```diff
@@ -63,7 +63,7 @@ export function handleLoginSuccess({
     SecureStorage.setItem("lastTokenSetTime", new Date().getTime());
 
     // send identify user event to telemetry to update the identity
-    telemetry.identify(resource.email || "");
+    resource.email && telemetry.identify(resource.email);
```

Comment:
I'm not a fan of using conditions like this as shorthand for an `if`. Might as well be verbose to make it a bit clearer.
```suggestion
    if (resource.email) {
      telemetry.identify(resource.email);
    }
```

or at very least use the single line shorthand
```suggestion
    if (resource.email) telemetry.identify(resource.email);
```

Reply:
updated